11LessonEleven/homework/css/mobile.css

```diff
@@ -0,0 +1,24 @@
+header {
+  background-image: url("../img/bg2.jpg");
+}
+
+p {
+  font-size: 25px;
+}
+
+header p {
+  margin: 30px 275px 0 0;
+}
+
+.blog, .food {
+  float:none;
+  padding: 40px 0 0 100px;
+}
+
+.blog p, .food p {
+  color: grey;
+}
+
+.content {
+  height: 600px;
+}
```

Comment:
instead of splitting your CSS into two seperate files, use media queries to specify styling for desktop vs mobile screen dimensions all within your `style.css` file